tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -369,10 +415,29 @@ struct lkl_netdev *lkl_netdev_dpdk_create(const char *ifparams, int offload,
 	/* we always enable big_packet mode with dpdk. */
 	nd->offload = offload;
 
-	snprintf(poolname, RTE_MEMZONE_NAMESIZE, "%s%s", "tx-", ifparams);
+	ifparams_len = strlen(ifparams);
```

Comment:
could you document somewhere (maybe https://github.com/lkl/linux/wiki/Howto:-DPDK-with-LKL should be good).  at least in the code comment block *and* commit message will be helpful.